ninja-jaxy-routes/src/main/java/ninja/jaxy/JaxyRoutes.java

```diff
@@ -217,14 +215,21 @@ public int compare(Method m1, Method m2) {
         return methods;
     }
 
+    /**
+     * Configures the set of packages to scan for annotated controller methods.
+     */
     private void configureReflections() {
-        ConfigurationBuilder builder = new ConfigurationBuilder();
-
-        Set<URL> packagesToScan = getPackagesToScanForRoutes();
-        builder.addUrls(packagesToScan);
-
-        builder.addScanners(new MethodAnnotationsScanner());
-        reflections = new Reflections(builder);
+        Optional<String> basePackage = Optional.of(ninjaProperties.get(NinjaConstant.APPLICATION_MODULES_BASE_PACKAGE));
```

Comment:
I think it should be ofNullable... otherwise we'll get an exception if the property is not defined...